integration/fuse/bin/alluxio-fuse

```diff
@@ -22,12 +22,13 @@ readonly MOUNT_USAGE="
 Mounts a path in Alluxio namespace (defaults to \"alluxio.fuse.mount.point\") 
 to mount point on local file system (defaults to \"alluxio.fuse.mount.alluxio.path\").
 
-Usage:\talluxio-fuse mount [-n] [-o <mount option>] [mount_point] [alluxio_path]
+Usage:\talluxio-fuse mount [-n] [-o <mount option>] [-t <sleep seconds>] [mount_point] [alluxio_path]
 \talluxio-fuse mount
 
 -o \tmount options for the fuse daemon,separated by comma
 -n \tno-daemon. This launches the process in the foreground and logs to stdout
--s \tstackFS. This launches StackFS process for testing purpose"
+-s \tstackFS. This launches StackFS process for testing purpose
+-t \tsleep time(seconds) before checking the AlluxioFuse process, default value is 2"
```

Comment:
before checking whether the AlluxioFuse process is alive.

Reply:
Thanks for the suggestion. I have fixed it.